src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -109,6 +109,12 @@ class LocationResource extends Resource {
                 return notFound().build()
             }
 
+            // validate ranges of lat and lon
+            if(lat && lon && !(lat >= -90 && lat <= 90 && lon >= -180 && lon <= 180)) {
```

Comment:
Could you add a spcae between `if` and the opening parentheses to match the styling of the other `if` statements in this class?